scripts/start

```diff
@@ -1,4 +1,14 @@
 #!/usr/bin/env sh
+if [ -z "$(docker network ls --filter name=^ffc-pay$ --format={{.Name}})" ]; then
+  echo "Creating ffc-pay Docker network"
+  docker network create ffc-pay
```

Comment:
do these docker scripts need to create the ffc-doc network instead of pay?

Reply:
This is the network that everything runs on together so it was left as ffc-pay